README.md

```diff
@@ -577,13 +578,30 @@ fetchContactsByPhone: (phoneNumbers: Array<string>) => Promise<Array<{
 }>>;
 ```
 
+### getAppMetadata
+
+Check if an app is installed in the phone
+
+-   Available for app versions 11.8 and higher
+
+```typescript
+getAppMetadata: (appToken: string) => Promise<{isInstalled: boolean; marketUrl: string; appUrl: string}>;
+```
+
+#### Example
+
+````javascript
+import {getAppMetadata} from '@tef-novum/webview-bridge';
+
+getAppMetadata('tokenAppToCheck').then(({isInstalled, marketUrl, appUrl}) => { ... });
```

Comment:
what is an `appToken`? Where do I get one?
what is an `appUrl`? How is that different from the `marketUrl`?

Please clarify that in the README